students/hyerim/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name = models.CharField(max_length=45)
+    email = models.CharField(max_length=100)
+    password = models.CharField(max_length=100)
+    mobile = models.CharField(max_length=100)
+    address = models.CharField(max_length=100)
+    birth_date = models.DateField()
+
+    class Meta:
+        db_table="user_info"
```

Comment:
```suggestion
class User(models.Model):
    name = models.CharField(max_length=45)
    email = models.CharField(max_length=100)
    password = models.CharField(max_length=100)
    mobile     = models.CharField(max_length=100)
    address    = models.CharField(max_length=100)
    birth_date = models.DateField()

    class Meta:
        db_table="users"
```
혜림님 변수명, Field 지정 다 너무 좋습니다! 예시로 수정한 것처럼 제일 긴 변수명 기준으로 = 정렬만 해주세요~
위코드 컨벤션에서 database 테이블명은 클래스명에서 소문자, 복수형태로 지정합니다.

Reply:
넵 확인했습니다

Reply:
연우님 settings.py 파일을 수정하고 git add 하는 걸 깜빡했네요. settings.py 추가하고 model 변수명, 클래스 명 수정 후 commit 했습니다!